ingests/un_wpp.py

```diff
@@ -0,0 +1,147 @@
+    ],
+    "demographics": [
+        "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/CSV_FILES/WPP2022_Demographic_Indicators_Medium.zip",
+        "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/CSV_FILES/WPP2022_Demographic_Indicators_OtherVariants.zip",
+    ],
+    "population": [
+        "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/CSV_FILES/WPP2022_PopulationBySingleAgeSex_Medium_1950-2021.zip",
+        "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/CSV_FILES/WPP2022_PopulationBySingleAgeSex_Medium_2022-2100.zip",
+        "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/CSV_FILES/WPP2022_PopulationBySingleAgeSex_High_2022-2100.zip",
+        "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/CSV_FILES/WPP2022_PopulationBySingleAgeSex_Low_2022-2100.zip",
+    ],
+    "deaths": [
+        "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/EXCEL_FILES/4_Mortality/WPP2022_MORT_F01_1_DEATHS_SINGLE_AGE_BOTH_SEXES.xlsx",
+        "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/EXCEL_FILES/4_Mortality/WPP2022_MORT_F01_2_DEATHS_SINGLE_AGE_MALE.xlsx",
+        "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/EXCEL_FILES/4_Mortality/WPP2022_MORT_F01_3_DEATHS_SINGLE_AGE_FEMALE.xlsx",
+    ],
+    "dependency_ratio": [
+        "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/EXCEL_FILES/2_Population/WPP2022_POP_F07_1_DEPENDENCY_RATIOS_BOTH_SEXES.xlsx",
+        "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/EXCEL_FILES/2_Population/WPP2022_POP_F07_2_DEPENDENCY_RATIOS_MALE.xlsx",
+        "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/EXCEL_FILES/2_Population/WPP2022_POP_F07_3_DEPENDENCY_RATIOS_FEMALE.xlsx",
+    ],
+}
+
+
+def download_data(output_dir):
+    """Download all data."""
+    log.info("Downloading data...")
+    for category, urls in URLS.items():
+        t0 = time.time()
+        log.info(category)
+        for url in urls:
+            filename = os.path.basename(url)
+            log.info(f"\t {filename}")
+            output_path = os.path.join(output_dir, filename)
+            _download_file(url, output_path)
+        t = time.time() - t0
+        log.info(f"{t} seconds")
+        log.info("---")
+
+
+def unzip_data(output_dir):
+    """Unzip downloaded files (only compressed files)."""
+    log.info("Unzipping data...")
+    files = [os.path.join(output_dir, f) for f in os.listdir(output_dir)]
+    for f in files:
+        log.info(f)
+        if f.endswith(".zip"):
+            _unzip_file(f)
+
+
+def _download_file(url, output_path):
+    """Download individual file."""
+    response = requests.get(url, stream=True)
+    with open(output_path, "wb") as f:
+        for chunk in response.iter_content(chunk_size=512):
```

Comment:
I wonder how fast this is, we've found `chunk_size=1024 * 1024 * 10` to be much faster for UN SDG.